### ISSUE TITLE
Code and bold markup not interchangeable

### ISSUE THREAD

**Issue by suedunham** [saschaleib/dokuwiki-template-ad-hominem#22]
When mixing `''` and `**` in the markup, the order used makes a difference in the output. This is not the case in the Default template. Presumably, `//` and `__` may not work as interchangeably as well, though I haven't looked for this.

For instance, these two lines have the same output in the Default template, but the main font is used for the first in Ad Hominem.

```
%%''**1 + 1 * 2 = 3**''%% gives ''**1 + 1 * 2 = 3**''\\
%%**''1 + 1 * 2 = 3''**%% gives **''1 + 1 * 2 = 3''**\\
```

**Comment by saschaleib** [comment]
Hello and thanks for reaching out. I am not sure if I understand what you man, though. If I try the examples you give, I get the following results:

- `''**1 + 1 * 2 = 3**''` ⟶  `<code><strong>1 + 1 * 2 = 3</strong></code>`
- `**''1 + 1 * 2 = 3''**` ⟶ `<strong><code>1 + 1 * 2 = 3</code></strong>`

Which is what I would consider the expected result. Also they don't visually appear any different, as the formatting of `<code>` and `<strong>` don't overlap.

Could you explain a bit more what would be your expected output, or how it should look for you? You can also post screenshots here, or a link to your site, where I can inspect the CSS in more detail.

Best greetings /s.

**Comment by saschaleib** [comment]
This turned out much easier than I thought. It is fixed in the repo now and will be available in the next update. If you need it urgently, you can just patch the CSS file "content.less" which contains this and some other fixes.

**Comment by saschaleib** [comment]
This is now fixed in the new versioon 1.4.

**Comment by suedunham** [comment]
Ah, excellent. Thank you for your work. Now back to trying to get my own
CSS to work properly with it. I'm still learning. :-D
